### DISCUSSION TITLE
Why are the "colour.XYZ_to_Oklab" and "colour.Oklab_to_XYZ" definitions shown on the home page not in the Conda package?

### DISCUSSION THREAD

**Discussion by abnormally-distributed** [colour-science/colour#875] [Q&A] [answered]
The function XYZ_to_Oklab shown ont he readme.txt on the github page doesn't seem to exist in the colour-science package. 

Here is the line showing the version of colour-science installed from `conda list`

`colour-science            0.3.16             pyh44b312d_1    conda-forge`

and attached is a picture showing the absence of it where  it should be in the list

![OKLab](https://user-images.githubusercontent.com/24922749/133708624-395a9b10-f1cc-44fc-8e20-5ef80b70ea2f.png)
![OKLab2](https://user-images.githubusercontent.com/24922749/133708623-5dc24889-f73b-4884-bd5e-6f75f5b8f0ac.png)

**Comment by thomasmansencal** [comment, marked as answer]
Hi @abnormally-distributed,

The definitions are available in the develop branch which is not released yet!

```python
>>> colour.XYZ_to_Oklab
<function XYZ_to_Oklab at 0x1204813a0>
```
